extension-androidauto/src/main/java/com/mapbox/maps/extension/androidauto/MapboxCarMap.kt

```diff
@@ -140,7 +159,7 @@ class MapboxCarMap {
    * interface, or override the [DefaultMapboxCarMapGestureHandler], or set to null to disable
    * gesture handling.
    */
-  fun setGestureHandler(gestureHandler: MapboxCarMapGestureHandler?) {
+  fun setGestureHandler(gestureHandler: MapboxCarMapGestureHandler?) = apply {
```

Comment:
do we really need to use apply to return the `MapbboxCarMap` after the execution? Would there be any use case?

Reply:
The apply exists on `registerObserver` and I was using it downstream to chain the calls

```
val map = MapboxCarMap().registerObserver(observer1)
  .registerObserver(observer1)
  .registerObserver(observer1)
```

But you can't add `setGestureHandler`